src/core/parser.ml

```diff
@@ -467,6 +467,18 @@ let term = term PFunc
 let parser rule =
   | l:term "↪" r:term -> Pos.in_pos _loc (l, r)
 
+(** [unif_rule] is a parser for unification rules. *)
+let parser unif_rule =
+  | l:{term "≡" term} "↪" r:{term "≡" term} rs:{"," term "≡" term}* ->
+      let equiv = Pos.none (P_Iden(Pos.none ([], "#equiv"), true)) in
+      let eqlist = Pos.none (P_Iden(Pos.none([], "#list"), true)) in
+      let mkequiv (l, r) =
+        Pos.none (P_Appl(Pos.none (P_Appl(equiv, l)), r))
+      in
+      let lhs = mkequiv l in
+      let rhs = p_add_args eqlist (List.map mkequiv (r::rs)) in
```

Comment:
Here, for better printing later, I would have 2 cases:
- rs = []: rhs = mkequiv r
- r=r1, rs=r2,..,rn<>[]: rhs = mkcomma s1 (mkcomma s2 ... (mkcomma s(n-1) sn) ...) where si = mkequiv ri